### ISSUE TITLE
Any reason scrollElement needs an ID?

### ISSUE THREAD

**Issue by samselikoff** [dollarshaveclub/ember-router-scroll#77]
Could we change scrollElement to accept any CSS selector, like a class?

**Comment by briangonzalez** [comment]
Yes, absolutely @samselikoff. Not sure that is the way it is. PR's welcomed.

**Comment by yowainwright** [comment]
It seems this issue was [fixed](https://github.com/dollarshaveclub/ember-router-scroll/commit/948f25d0d5e27f0a6608207e50f688d6c0ed6bc1#diff-04c6e90faac2675aa89e2176d2eec7d8).

**Comment by samselikoff** [comment]
@yowainwright I believe this is still an issue: https://github.com/dollarshaveclub/ember-router-scroll/blob/master/addon/services/router-scroll.js#L54

I'd like `scrollElement` to be able to be any css selector (not targetElement)

Mind reopening?

**Comment by yowainwright** [comment]
@samselikoff do you have time to submit a PR with this update? Thank you. 

Thanks for articulating the issue as well.

**Comment by samselikoff** [comment]
I don't at the moment but let's leave this open for now, may be able to carve out some time soon-ish
